python/tools/build_manylinux_wheels.sh

```diff
@@ -23,10 +23,10 @@ pushd /code/python
 rm -rf wheels dist build
 for py in "${py_versions[@]}"
 do
-  /opt/python/${py}-${py}/bin/pip install numpy "pyarrow<9" cython
+  /opt/python/${py}-${py}/bin/pip install numpy "pyarrow>=9,<10" cython
```

Comment:
can you change the thirdparty build script to checkout the right version?

Reply:
🆗